Formula/mapnik.rb

```diff
@@ -33,13 +33,6 @@ class Mapnik < Formula
   depends_on "proj"
   depends_on "webp"
 
-  # Fix for Boost >= 1.75
-  # https://github.com/mapnik/mapnik/issues/4201
-  patch do
-    url "https://github.com/mapnik/mapnik/commit/49e0ef18.patch?full_index=1"
-    sha256 "d8f12a85ad78f95e3cb2b3b5485e586c250fe2230a90874c0a70843189cc42f5"
-  end
-
   def install
     ENV.cxx11
```

Comment:
These lines don't do anything anymore, do they?

`mapnik` apparently requires c++14, and seems to build fine despite this.